xray/test/test_plot.py

```diff
@@ -284,6 +404,13 @@ def test_extend(self):
         artist = self.plotmethod(vmin=-10, vmax=0)
         self.assertEqual(artist.extend, 'max')
 
+    def test_levels(self):
```

Comment:
Not exactly sure what this does since the `plotmethod` doesn't seem to have a levels arg at the moment.